example/lib/main.dart

```diff
@@ -67,6 +67,18 @@ class _AtlasSampleState extends State<AtlasSample> {
       )
     ],
   );
+  final Set<Circle> _circles = Set<Circle>.from(
```

Comment:
There's no need for the initial type, you can write this as:

```dart
final _circles = Set<Circle>.from(...)
```

If you make this change, please update the README as well